docs/user/alerting/action-types/pagerduty.asciidoc

```diff
@@ -4,19 +4,152 @@
 
 The PagerDuty action type uses the https://v2.developer.pagerduty.com/docs/events-api-v2[v2 Events API] to trigger, acknowledge, and resolve PagerDuty alerts.
 
+* <<pagerduty-benefits, PagerDuty and Elastic integration benefits>>
+* <<pagerduty-connector-configuration, Connector configuration>>
+* <<pagerduty-action-configuration, Action configuration>>
+
+[float]
+[[pagerduty-benefits]]
+=== PagerDuty and Elastic integration benefits
+
+By integrating PagerDuty with Elastic Alerts, you can:
+
+* Route your Elastic Alerts to the right PagerDuty responder within your team, based on your structure, escalation policies, and workflows.
+* Automatically generate incidents of different types and severity based on each alert’s context.
+* Tailor the incident data to match your needs by easily passing the alerting context from Kibana to PagerDuty.
+
+[float]
+[[pagerduty-how-it-works]]
+=== How it works
+
+{kib} allows you to create threshold alerts of a generic type that you can use for all your indices.
+You can also create specific alerts for Observability and SIEM.
+Create an alert to be notified of a significant move in your datasets. Elastic Alerts will generate PagerDuty incidents.
+
+. To connect your {kib} alerts to PagerDuty incidents, create a PagerDuty Connector in Kibana.  You can:
++
+* Create a connector as part of creating an alert by selecting PagerDuty in the *Actions*
+section of the alert configuration and selecting *Add new*.
+* Alternatively, create a connector by navigating to Management from the {kib} navbar and selecting
+*Alerts and Actions*. Then select the *Connectors* tab, click on the *Create connector* button, and select the PagerDuty option.
+
+. Enter a name for the connector, the API URL, and the Routing Key.
```

Comment:
The API URL seems to be optional. Maybe we call that out what the default is here?